### PR TITLE
set default value for option --leader-election-id

### PR DESCRIPTION
**Pull request by llamerada-jp**
Signed-off-by: Yuji Ito <llamerada.jp@gmail.com>

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -11,6 +11,7 @@ This file itself is based on [Keep a CHANGELOG](https://keepachangelog.com/en/0.
 
 ### Changed
 - Upgrade for Kubernetes 1.17 (#115).
+- topolvm-controller requires `--leader-election-id` flag.
```
Comment (ymmt2005) [inline CHANGELOG.md:14]: No longer required as it has a default value.

Comment (llamerada-jp) [reply CHANGELOG.md:14]: TopoLVM v0.4.1 requires `--leader-election-id` and v0.4.2(next) set default value on it.

Comment (ymmt2005) [reply CHANGELOG.md:14]: I see.

Comment (dulltz) [reply CHANGELOG.md:14]: @ymmt2005 
Do you mean that we do not need to tweak the changelog v0.4.1?

Comment (ymmt2005) [reply CHANGELOG.md:14]: ?  This was resolved already.

Comment (dulltz) [reply CHANGELOG.md:14]: We are going to include this fix in v0.4.2.

Comment (dulltz) [reply CHANGELOG.md:14]: @ymmt2005 

> ? This was resolved already.

Do you mean that this PR is LGTM?

Comment (ymmt2005) [reply CHANGELOG.md:14]: Not actually took a look.  But LGTM.

**Pull request merged**